modules/ref_device_module/src/ref_device_impl.cpp

```diff
@@ -121,8 +121,8 @@ void RefDeviceImpl::initClock()
 
 void RefDeviceImpl::initIoFolder()
 {
-    aiFolder = this->addIoFolder("ai", ioFolder);
-    canFolder = this->addIoFolder("can", ioFolder);
```

Comment:
I suppose these are okay for now as these are reference modules. But in general we need to avoid any changes to the local IDs of our components, as we'll break every stored setup doing so.

I don't think I noticed any local Id changes throughout the PR except in the reference modules, but if there are any, they should be kept as-is.